script/Arcueid/Day 11/11_00_ARC11_3.txt

```diff
@@ -312,7 +312,7 @@ Arcueid suddenly looks up, as though something has caught her attention.
 [sha:fd9001c98a4650ea68888f45bbfbbb241d99bcac]{
 -- Page 175, Offset 30007.
 -- 「志貴、いつまでも帰らないんだもの。
-"Shiki, you just won't go home, will you?
+"Shiki, you just won't go home, will you.
```

Comment:
is this change inspired by the VA? If not, revert

Reply:
hard to tell, will do